crates/generate_artifacts/src/entrypoint_artifact.rs

```diff
@@ -176,15 +189,36 @@ pub(crate) fn generate_entrypoint_artifacts_with_client_field_traversal_result<
         },
     );
 
-    let mut paths_and_contents = EntrypointArtifactInfo {
-        query_text,
-        query_name,
-        parent_type: parent_object,
-        normalization_ast_text,
-        refetch_query_artifact_import,
-        concrete_type: concrete_type.name,
+    let mut paths_and_contents = vec![];
+    let persist = persisted_documents_options.is_some();
+    if !persist {
```

Comment:
I'm a bit torn here.

On the one hand, I think it's worth generating the query text, even if it isn't used, because it is nice to see that what effect a change to an isograph literal had, and query texts have `format::pretty`, so its easy to see what changed.

On the other hand, that might be unexpected behavior.

Anyway, I think it's best to always generate the query. Perhaps it makes sense to generate a `.sql` or `.graphql` file with the query always, instead of something that exports a string, but anyway, for now, lets always generate the `query_text.ts` file